app/api/events.py

```diff
@@ -37,7 +81,7 @@ class EventList(ResourceList):
 
 
 class EventRelationship(ResourceRelationship):
-    decorators = (jwt_required, )
+    decorators = (is_user_itself, )
```

Comment:
Use jwt_required here for now, is_user_itself is for user_id related changes only